src/ducks/search/SearchHeader.jsx

```diff
@@ -24,17 +26,30 @@ const SearchHeader = ({
 }) => {
   const { isMobile } = useBreakpoints()
   const { t } = useI18n()
+  const {
+    isSelectionModeEnabled,
+    isSelectionModeActive,
+    setIsSelectionModeActive
+  } = useSelectionContext()
 
   const handleReset = inputNode => {
     setSearch('')
+    if (isSelectionModeEnabled) setIsSelectionModeActive(false)
```

Comment:
nit: Just `setIsSelectionModeActive(false)` is enough I think.

Reply:
non `isSelectionModeEnabled` c'est pour tester si le flag est actif, à ne pas confondre avec `isSelectionModeActive`

Reply:
Ah d'accord